File: BookStore.BuisneLogic/Models/PrintingEditionsFilterModel/PrintingEditionsFilterModel.cs
using BookStore.BusinessLogic.Models.Base;
using System.Collections.Generic;
using static BookStore.BusinessLogic.Models.Enums.Enums;

namespace BookStore.BusinessLogic.Models.PrintingEditionsFilterModel
{
    public class PrintingEditionsFilterModel : BaseFilterModel
    {
        public List<TypePrintingEdition> Categories { get; set; }
        public PrintingEditionSortType SortType { get; set; }
        public Currencys Currency { get; set; }
        public decimal MinPrice { get; set; }
        public decimal MaxPrice { get; set; }
    }
}

File: StoreClient/src/app/shared/models/PeintingEditions/printing-editions-filter-model.ts
import { PrintingEditionSortType } from 'src/app/shared/enums/printing-edition-sort-type';
import { TypePrintingEdition } from 'src/app/shared/enums/type-printing-edition';
import {BaseFilterModel} from '../Base/base-filter-model';
import { Currencys } from '../../enums/currencys';

export class PrintingEditionsFilterModel extends BaseFilterModel {
  categories: Array<TypePrintingEdition>;
  sortType: PrintingEditionSortType;
  minPrice: number;
  maxPrice: number;
  currency: Currencys;
}
